Zeeguu Reader/Zeeguu Reader/AppDelegate.swift

```diff
@@ -73,9 +84,26 @@ class AppDelegate: UIResponder, UIApplicationDelegate, UISplitViewControllerDele
 		self.window?.rootViewController?.presentViewController(vc, animated: true, completion: nil)
 	}
 	
+	func userLoggedIn(notification: NSNotification) {
+		let def = NSUserDefaults.standardUserDefaults()
+		if def.boolForKey(DidShowWelcomeScreenKey) == false {
+			def.setBool(true, forKey: DidShowWelcomeScreenKey)
+			def.synchronize()
+			
+			let vc = UINavigationController(rootViewController: WelcomeViewController())
+			vc.modalPresentationStyle = .FormSheet
+			self.window?.rootViewController?.presentViewController(vc, animated: true, completion: nil)
+		}
+	}
+	
 	func applicationWillResignActive(application: UIApplication) {
 		// Sent when the application is about to move from active to inactive state. This can occur for certain types of temporary interruptions (such as an incoming phone call or SMS message) or when the user quits the application and it begins the transition to the background state.
 		// Use this method to pause ongoing tasks, disable timers, and throttle down OpenGL ES frame rates. Games should use this method to pause the game.
+		guard let date = becomesActiveDate else {
+			return
+		}
+		let interval = -date.timeIntervalSinceNow
```

Comment:
Ah! ok :)

On Thu, Jun 16, 2016 at 5:45 PM, JorritO <notifications@github.com> wrote:

> In Zeeguu Reader/Zeeguu Reader/AppDelegate.swift
> <https://github.com/mircealungu/Zeeguu-Reader-iOS/pull/7#discussion_r67369904>
> :
>
> >  	func applicationWillResignActive(application: UIApplication) {
> >  		// Sent when the application is about to move from active to inactive state. This can occur for certain types of temporary interruptions (such as an incoming phone call or SMS message) or when the user quits the application and it begins the transition to the background state.
> >  		// Use this method to pause ongoing tasks, disable timers, and throttle down OpenGL ES frame rates. Games should use this method to pause the game.
> > +		guard let date = becomesActiveDate else {
> > +			return
> > +		}
> > +		let interval = -date.timeIntervalSinceNow
>
> Because timeIntervaleSinceNow returns a negative number if date is
> history.
>
> So, if date is 2016-06-16 17:00 and 'now' is 2016-06-16 17:10, then
> timeIntervalSinceNow will return -600 seconds.
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/mircealungu/Zeeguu-Reader-iOS/pull/7/files/3cb573e63c3f7cf7afafe589a9606394dda3e446..0ed74e8535332054594876fa59af585439a5641f#r67369904>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAcWh3L4vcLM7Goe9RzFOsbdfnlqEGwoks5qMW-MgaJpZM4IvyI2>
> .
>